.github/workflows/integration_test_run.yaml

```diff
@@ -171,10 +175,26 @@ jobs:
         )) || inputs.runs-on
       }}
     steps:
+      - name: Setup Canonical k8s
+        if: ${{ inputs.use-canonical-k8s }} == true
+        id: canonical-k8s
+        run: |
+          sudo snap install k8s --channel=1.30-classic/beta --classic
```

Comment:
I think you can reuse the `channel` input here currently used for microk8s to be able to customise the snap channel